content/docs/replication/replication-kubevirt.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Replication with KubeVirt
+linktitle: Replication with KubeVirt
+weight: 6
+description: >
+  Support of Replication enabled CSI drivers with virtualized workloads.
+---
+
+Starting in v1.11.0, the Replication module supports the volume replication for
+Persistent Volumes (PVs) created via virtualized workloads (VMs).
+
+## KubeVirt:
+
+- KubeVirt allows virtual machines (VMs) to run alongside container workloads
+  within Kubernetes clusters, addressing the coexistence of legacy VM workloads
+  and modern containers.
+
+## KubeVirt support with Replication
+
+- Install the OpenShift Virtualization operator from the RedHat UI on both the
+  source and target clusters with OCP v4.18 and above.
+- The Replication modes supported are: SYNC and ASYNC.
+
+- Make sure the driver is installed with the Replication Module enabled and a

```

Comment:
Link to where the instructions are for enabling the replication module

Reply:
Addressed.